docs/js-client-sdk.eppojsclient.initialized.md

```diff
@@ -4,6 +4,10 @@
 
 ## EppoJSClient.initialized property
 
+> Warning: This API is now obsolete.

```

Comment:
did you mean `deprecated` instead of `obsolete`?

Reply:
I'm not sure where`yarn docs` is getting this text from

Reply:
I am slo not sure why I can't get `yarn docs` to pick up the latest code and not document this as deprecated.